airflow/www/utils.py

```diff
@@ -99,7 +103,42 @@ def wrapper(*args, **kwargs):
         session.commit()
 
         return f(*args, **kwargs)
-    
+
+    return wrapper
+
+
+def notify_owner(f):
+    '''
+    Decorator to notify owner of actions taken on their DAGs by others
+    '''
+    @functools.wraps(f)
+    def wrapper(*args, **kwargs):
+        if request.args.get('confirmed') == "true":
```

Comment:
Doesn't work in practice since for unconfirmed it just has nothing. (and not confirmed=false)